vhd/packer/configure-windows-vhd.ps1

```diff
@@ -81,6 +81,7 @@ function Get-FilesToCacheOnVHD
             "https://acs-mirror.azureedge.net/wink8s/v1.16.1-1int.zip",
             "https://acs-mirror.azureedge.net/wink8s/v1.16.2-1int.zip",
             "https://acs-mirror.azureedge.net/wink8s/v1.16.4-1int.zip",
+            "https://acs-mirror.azureedge.net/wink8s/v1.16.6-1int.zip",

```

Comment:
@marosset Do we want to keep all these Windows versions pre-downloaded into the VHD, even those that aren't supported in the supported list above?

Reply:
I think we usually just have supported versions here.

Reply:
It won't hurt anything tho, we have plenty of space on the VHD currently.